collect_app/build.gradle

```diff
@@ -63,7 +63,7 @@ android {
     defaultConfig {
         applicationId("org.odk.collect.android")
         minSdkVersion(16)
-        targetSdkVersion(22)
+        targetSdkVersion(25)
```

Comment:
Revert back the changes in another commit. Also, there are some merge conflicts. Could you please resolve them as well?